Pomodoro.Api/ViewModels/Frequency/FrequencyViewModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿// <copyright file="FrequencyViewModel.cs" company="PomodoroGroup_GL_BaseCamp">
+// Copyright (c) PomodoroGroup_GL_BaseCamp. All rights reserved.
+// </copyright>
+using Pomodoro.Core.Enums;
+using Pomodoro.Core.Models.Tasks;
+
+namespace Pomodoro.Core.Models.Frequency
```

Comment:
Pay attention that the namespaces of your view models don't match their locations.